manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.locale.de-DE.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182
+PackageLocale: de-DE
+Publisher: 34u GmbH
+PublisherUrl: https://www.multibasecs.de
+PublisherSupportUrl: https://www.multibasecs.de/hilfe/
+PrivacyUrl: https://www.multibasecs.de/impressum/
+PackageName: MultiBaseCS Home

```

```suggestion
PackageName: MultiBaseCS 5 Home
```
This should match control panel